iot-hub/Samples/device/DeviceReconnectionSample/DeviceReconnectionSample.cs

```diff
@@ -170,14 +170,14 @@ private void ConnectionStatusChangeHandler(ConnectionStatus status, ConnectionSt
                             _logger.LogWarning("### The DeviceClient has been disconnected because the retry policy expired." +
                                 "\nIf you want to perform more operations on the device client, you should dispose (DisposeAsync()) and then open (OpenAsync()) the client.");
 
-                            InitializeClient();
+                            await InitializeAndOpenClientAsync();
                             break;
 
                         case ConnectionStatusChangeReason.Communication_Error:
                             _logger.LogWarning("### The DeviceClient has been disconnected due to a non-retry-able exception. Inspect the exception for details." +
```

Comment:
I know this is already there but change to string interpolation?

Reply:
This is just a long string broken into two links, string interpolation wouldn't really help here.

Reply:
Ahhh I see!